cmd/batch-expire.go

```diff
@@ -131,13 +132,33 @@ var _ yaml.Unmarshaler = &BatchJobExpireFilter{}
 // UnmarshalYAML - BatchJobExpireFilter extends unmarshal to extract line, col
 // information
 func (ef *BatchJobExpireFilter) UnmarshalYAML(value *yaml.Node) error {
-	type expFilter BatchJobExpireFilter
-	var tmp expFilter
+	tmp := struct {
```

Comment:
If you have modified `(*BatchReplicateFilter).UnmarshalYAML` why do another translation here?

Wouldn't the original code work?